src/user/importexport-app.md

```diff
@@ -133,49 +132,104 @@ Import.
 import data produced by off-line PDF data entry forms. Please refer to
 the section **Data set management** for details on how to produce a PDF
 form which can be used for off-line data entry.
 
-To import a PDF data file, navigate to the *PDF Data Import* item in the
-side menu. Upload the completed PDF file and click *Import*.
+To import a PDF data file, navigate to the _PDF Data Import_ item in the
+side menu. Upload the completed PDF file and click _Import_.
 
-### Event Import { #event_import } 
+### Event Import { #event_import }
 
-Event can be access from the sidebar by clicking on Event
-import.
+Event Import can be accessed from the sidebar by clicking on Event
+Import.
 
 ![](resources/images/import_export/event_import.png)
 
-1.  Select a format: *JSON* , *CSV*, or *XML*
+1.  Select a format: _JSON_ , _CSV_, or _XML_
 
 2.  Click **Advanced options** if you want to adjust one or more of
     the following settings before importing:
 
-    * Event ID scheme
-    * Data element ID scheme
-    * Org unit ID scheme
-    * ID scheme
+    - Event ID scheme
+    - Data element ID scheme
+    - Org unit ID scheme
+    - ID scheme
 
 3.  Click on the **Import** button which will upload the file and start the
     importing process.
 
-### GML Import { #gml_import } 
+### Earth Engine Import { #ee_import }
 
-GML Import can be accessed from the sidebar by clicking on GML
+Earth Engine Import can be access from the sidebar by clicking on Earth Engine
 Import.
 
-![](resources/images/import_export/gml_import.png)
+Google Earth Engine (WorldPop) data is fetched using the Google Earth Engine API. A Google
+EarthEngine account is required to use this importer.
+
+![](resources/images/import_export/ee_import.png)
+
+#### Select which Earth Engine data should be imported
+
+The first section of the form is used to configure the Earth Engine data to import.
+
+1. Select which Earth Engine dataset should be imported. Currently the choices are _Population_ and _Population age groups_.
+
+2. After a dataset has been selected, a period must be selected. Only one period can be imported at a time.
+
+3. Choose how the data should be rounded. By default data is not rounded.
+
+4. Select which organisation units to import data for. If you select facility level organisation units, then you must choose an associated geometry for the facilities. Without an associated geometry for facilities, the Earth Engine cannot determine the population.
+
+![](resources/images/import_export/ee_ou_associated_geometry.png)
+
+#### Select the data elements to import the Earth Engine data into
+
+Once you have configured the Earth Engine dataset, you need to select which data element to import the data into. For datasets with disaggregation groups, such as "Population age groups", the DHIS2 data element must have disaggregations in the form of category option combos that match the Earth Engine dataset disaggregation groups.
```

Comment:
"to import the data into" also sounds a bit Norwegian English to me :-)

Reply:
I think I've lived here too long 😛 I don't know my prepositions anymore. Also, from a pedantic standpoint sentences should not end with a preposition, but I hear that is outdated thinking 🤷

Reply:
I think it sounds okay.